app-devel/lazygit/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=lazygit
+PKGSEC=devel
```

Comment:
`git` [is](https://github.com/AOSC-Dev/aosc-os-abbs/blob/597a7ab29c0c74560725e86898cf2903996c59f4/app-vcs/git/autobuild/defines#L2) in the `vcs` section, maybe move this package to the same section?
```suggestion
PKGSEC=vcs
```

Reply:
Yes, I have moved the package.